web-marketplace/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -43,6 +45,7 @@ import {
 import { useProfileItems } from './hooks/useProfileItems';
```

Comment:
missing translation for `profileVariantMapping`

Reply:
The variable holds variants meant to customize a component; it's not meant to be displayed.

Reply:
oh right!